tests/api.c

```diff
@@ -36922,6 +37006,10 @@ void ApiTest(void)
     test_wc_PKCS7_Degenerate();
     test_wc_PKCS7_BER();
     test_PKCS7_signed_enveloped();
+    AssertIntEQ(test_wc_PKCS7_NoDefaultSignedAttribs(), 0);
```

Comment:
can we please move the assert's up into the test function itself? It's easier debugging issues when the assert gives the exact line of the issue. See test_PKCS7_signed_enveloped function for example.